src/verso/Verso/Doc/Elab/Monad.lean

```diff
@@ -14,6 +14,7 @@ import Lean.DocString.Syntax
 import Verso.Doc
 import Verso.Doc.ArgParse
 import Verso.Doc.Elab.InlineString
+import Verso.Doc.Elab.Types
```

Comment:
Normally we'd call a module like this `.Basic` instead of `.Types` (especially because it doesn't seem to be mostly datatype definitions, but rather a mix of datatypes and operations on them)